task_6/1.js

```diff
@@ -0,0 +1,23 @@
+// запрос на покемон, получить urls всех покемонов, получить массив объектов покемонов по urls
+
+const getPokemons = async (url) => {
+    const pokemonsArray = [];
+    const pokemonsUrl = [];
+
+    try {
+        const {results} = await fetch(url).then((response) => response.json());
+        results.forEach(({url}) => pokemonsUrl.push(url));
```

Comment:
this is redundant. you can use map method. No need to collect urls in separate variable